app/controllers/application_controller.rb

```diff
@@ -35,6 +35,14 @@ def authenticate_user!
     end
   end
 
+  def user_banished?(user_or_org)
```

Comment:
We could move this to `models/user.rb` as `def banished?`. We can have orgs return false for the same method. After all this is not exactly  "controller logic"

Reply:
Changed ✅